templates/default/mesos-init.erb

```diff
@@ -1,6 +1,6 @@
-description "mesos <%= @type %>"
+description "Upstart job for <%= @wrapper %>"
```

Comment:
Yes, this is one of the major reasons I have created the binary wrapper outside the upstart job. Btw, don't forget Debian 8 :)